gcc-6-32bit.cmake

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2013-2017, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_GCC_32BIT_CMAKE)
+  return()
+else()
+  set(POLLY_GCC_32BIT_CMAKE 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+polly_init(
+    "gcc 6 / c++14 support / 32 bit"
+    "Unix Makefiles"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/compiler/gcc-6.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx14.cmake")
```

Comment:
gcc toolchains by default comes with C++11 (e.g. `gcc-7`), if you want to have C++14 please rename it to `gcc-6-32bit-cxx14`.